provisioning/Build.ps1

```diff
@@ -61,6 +61,9 @@ If ($mainDir -eq "C:\vagrant") {
     $npmInstallDir = (get-item $originalBuildScriptPath).parent.FullName
 }
 
+# Build the tray button
+../trayButton/build.ps1
```

Comment:
I'm wondering whether we should build the native tray button as a postinstall action rather than from a provisioning script. Same applies for generating the language bundle. @stegru @amb26 @danailbd what do you think?

Reply:
Yes, I think it's fine, you can just call the build script from the npm install (or postinstall) script like this:

`"postinstall": "npm run build"`

and leave the "build" npm script as you have it already (compiles messages and build the native tray).
What I want to avoid here is to end up without the tray after running npm install (we don't always run the provisioning scripts)